Week3/homework/step2-6.js

```diff
@@ -4,13 +4,21 @@ const arr2d = [[1, 2], [3, 4], [5, 6]];
 const arr3d = [[[1, 2], [3, 4]], [[5, 6], [7, 8]]];
 
 function printArray2d(arr) {
-  // Replace this comment and the next line with your code
-  console.log(arr);
+  for (let i = 0; i < arr.length; i++) {
+    for (let b = 0; b < arr[i].length; b++) {
```

Comment:
It is customary to use the letters `i`, `j`, `k`, `m` and `n` for nested loops (as you would in algebra). Note the letter `l` is to be avoided as it easily confused with the digit `1`.